src/components/CompareResults/beta/NoResultsFound.tsx

```diff
@@ -16,7 +16,7 @@ function NoResultsFound() {
   }; 
 
   return (
-    <Container>
+    <Container data-testid='no-results-found-msg'>
       <Grid container alignItems='center' justifyContent='center'>
```

Comment:
We need to add `paddingTop: Spacing.xxLarge` here, so that the magnifying glass is a bit lower.